src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -358,6 +358,18 @@ protected boolean parseArgs(final String[] argv) {
         return true;
     }
 
+    private CommandLineParser determineArgParser(String[] argv) {
+        // If there are args that don't start with dashes and contain equals then we use the old argument parser
+        boolean hasEqualsNoDash = Arrays.stream(argv).anyMatch(arg -> arg.contains("=") && !arg.startsWith("-"));
+
+        commandLineParser = (hasEqualsNoDash) ?
+                new LegacyCommandLineArgumentParser(this) :
+                new CommandLineArgumentParser(this,
+                        Collections.emptyList(),
+                        new HashSet<>(Collections.singleton(CommandLineParserOptions.APPEND_TO_COLLECTIONS)));
```

Comment:
`Collections.singletion()` already is a set (and `CommandLineArgumentParser` doesn't modify its arguments)  so the `new HashSet<>()` is unnecessary
```suggestion
                       Collections.singleton(CommandLineParserOptions.APPEND_TO_COLLECTIONS));
```